tests/scripts/format.py

```diff
@@ -70,11 +74,16 @@ def main(argv):
         if not os.path.isdir(dotnetcliPath):
             raise
 
-    if platform == 'Linux' or platform == 'OSX':
-        dotnetcliUrl = "https://go.microsoft.com/fwlink/?LinkID=809118"
+    print("Downloading dotnetcli")
```

Comment:
Since this is part of a human interface, can we call it the more consistent ".NET CLI" or ".NET Command Line Interface"?